docs/source/60_building_apps.md

```diff
@@ -256,6 +256,31 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s RegionJoin API
```

Comment:
My general thinking was that `70_algorithms.md` is for the detailed implementation notes (e.g., "to optimize join performance, we use an XYZ tree with an ABC load balancer.") while this is more intended as documentation for someone building an app using the ADAM APIs. I do agree though; I think this "chapter" reads right now as more of a "pull in these dependencies", less "here is how to use the APIs". Perhaps we should split the chapter?